roles/common/files/sensu_plugins/metrics-neutron-fip.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
I don't know enough about the differences between the checks and metrics and its not worth the time right now to figure that out.